doc/get_started/ubuntu-acp-1104-c.md

```diff
@@ -0,0 +1,137 @@
+---
+platform: ubuntu
+device: acp-1104
+language: c
+---
+
+Run a simple C sample on ACP-1104 device running Ubuntu
+===
+---
+
+# Table of Contents
+
+-   [Introduction](#Introduction)
+-   [Step 1: Prerequisites](#Prerequisites)
+-   [Step 2: Prepare your Device](#PrepareDevice)
+-   [Step 3: Build and Run the Sample](#Build)
+-   [Tips](#tips)
+
+<a name="Introduction"></a>
+# Introduction
+
+**About this document**
+
+This document describes how to connect **ACP-1104** device running **Ubuntu** with Azure IoT SDK. This multi-step process includes:
+-   Configuring Azure IoT Hub
+-   Registering your IoT device
+-   Build and deploy Azure IoT SDK on device
+
+<a name="Prerequisites"></a>
+# Step 1: Prerequisites
+
+You should have the following items ready before beginning the process:
+
+-   [Prepare your development environment][setup-devbox-linux]
+-   [Setup your IoT hub][lnk-setup-iot-hub]
+-   [Provision your device and get its credentials][lnk-manage-iot-hub]
+
+<a name="PrepareDevice"></a>
+# Step 2: Prepare your Device
+-   Follow the instructions from [device website](http://www.aaeon.com/en/p/infotainment-multi-touch-panel-solutions-1104/).
+
+<a name="Build"></a>
+# Step 3: Build and Run the sample
+
+<a name="Load"></a>
+## 3.1 Build SDK and sample
+
+-   Open a PuTTY session and connect to the device.
+
+-   Install the prerequisite packages for the Microsoft Azure IoT Device SDK for C by issuing the following commands from the command line on your board:
+
+        sudo apt-get update
+
+        sudo apt-get install -y curl libcurl4-openssl-dev uuid-dev uuid g++ make cmake git unzip openjdk-7-jre
+    
+    ***Note:*** *This setup process requires cmake version 3.0 or higher.* 
```

Comment:
Same comments like for the other docs (cmakeversion, dependencies, align with Dominic's latest changes).